packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -1073,4 +1145,146 @@ The Authenticator object defines the following properties:
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Authenticator Method object
+
+#### Authenticator Method properties
+
+| Property | Type | Description  | Applies to Authenticator Method type |
+| -------- | ---- | ------------ | ------------------------------------ |
+| `status` | String (Enum) | The status of the authenticator method. Possible values: `ACTIVE` or `INACTIVE` | All |
```

Comment:
Maybe chance the "All" to "All Authenticators" or "All Methods" to match the Authenticator properties table.

Reply:
Replaced "All" with "All Authenticator Methods". Thanks for the suggestion.